### PR TITLE
Dg update2

### PR DESCRIPTION
**Pull request by Asuraxsoul**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1016,31 +1056,89 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `delete -c 1`<br>
+      Expected: First `Consultation` is deleted from the list. Details of the deleted `Consultation` shown in the status message. GUI `Consultation` list is immediately updated.
+
+   1. Test case: `delete -c 0`<br>
+      Expected: No `Consultation` is deleted. Error details shown in the status message. GUI `Consultation` list under `Consultation` tab remains the same.
+
+   1. Other incorrect delete commands to try: `delete`, `delete -c x`, `...` (where x is larger than the list size or smaller than 1)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
```
Comment (wireseo) [inline docs/DeveloperGuide.md:1038]: Thanks for removing these!

File: docs/diagrams/AddCommandClassDiagram.puml [hunk 2]
```diff
@@ -5,8 +5,8 @@ hide circle
 
 class AddXYZCommand {
     /' Attributes '/
-    + MESSAGE_SUCCESS: String
-    + MESSAGE_INVALID: String
+    <u>+ MESSAGE_SUCCESS: String
```
Comment (wireseo) [inline docs/diagrams/AddCommandClassDiagram.puml:8]: Good note of static variables!

File: docs/diagrams/AddTaskSequenceDiagram.puml [hunk 3]
```diff
@@ -33,7 +31,7 @@ AddressBookParser -> AddCommandParser : parse("-e...")
 activate AddCommandParser
 
 activate TaskCommandParser
-AddCommandParser -> TaskCommandParser : parse("-e...")
+AddCommandParser -> TaskCommandParser : parseEvent("-e...")
```
Comment (wireseo) [inline docs/diagrams/AddTaskSequenceDiagram.puml:34]: For Add Tasks do we not need a ParserUtil class?

Comment (Asuraxsoul) [reply docs/diagrams/AddTaskSequenceDiagram.puml:34]: Ok I will add it in the next pr, thanks for mentioning it

**Pull request merged**